packages/send-emails/backend/python/app.py

```diff
@@ -84,7 +80,7 @@ def build_auth_url():
     auth_url = nylas.authentication_url(
         (CLIENT_URI or "") + request_body["success_url"],
         login_hint=request_body["email_address"],
-        scopes=DEFAULT_SCOPES,
+        scopes=['email.send', 'email.modify'],
```

Comment:
Should use `WebhookTriggers` enum

```js
        scopes=[WebhookTriggers.EmailSend, WebhookTriggers.EmailModify],
```

Make sure to also import:

```js
const { WebhookTriggers } = require('nylas/lib/models/webhook');
```

Reply:
forgive me i am exhausted

Reply:
wrong